src/app/client/src/app/modules/learn/components/course-consumption/assessment-player/assessment-player.component.ts

```diff
@@ -423,6 +426,17 @@ export class AssessmentPlayerComponent implements OnInit, OnDestroy, ComponentCa
     if (request.status === 2 && !this.isUnitCompleted) {
       this.logAuditEvent();
     }
+    const currentContent  = this.getActiveContent();
+    const summary = _.get(telObject, 'edata.summary');
+      let totallength;
+      summary?.forEach((k) => {
+        if (k['totallength']) {
+          totallength = k['totallength'];
+        }
+    });
+    if ((_.get(currentContent, 'mimeType') === 'application/pdf') && eid === 'END' && totallength === 1) {
```

Comment:
did you verified why the above `const request` object status is failing for PDF?

> status: (eid === 'END' && (_.get(this.getCurrentContent, 'contentType') !== 'SelfAssess') && this.courseProgress === 100
      && !this.isStatusChange) ? 2 : 1,

Even if it is failing, we could have added check similar to H5P content 

> // Set status to 2 if mime type is application/vnd.ekstep.h5p-archive
    if (isMimeTypeH5P) {
      request['status'] = 2;
    }

This will make sure the changes specific to PDF only.

End event summary has the details on clicking on back button

> "summary": [{"progress":100},{"totallength":1},{"visitedlength":1},{"visitedcontentend":true},{"totalseekedlength":0},{"endpageseen":false}]

Reply:
@vinukumar-vs 
status: (eid === 'END' && (_.get(this.getCurrentContent, 'contentType') !== 'SelfAssess') && this.courseProgress === 100
&& !this.isStatusChange) ? 2 : 1, this condition is failing as we click on the back button the this.isStatusChange will be true

Reply:
@vinukumar-vs  we cannot add condition like H5P as h5P if it is open even it is multiple page we consider as 100% progress 
but this is only with single page PDF